client/app/instructor/dashboard/dashboard.html

```diff
@@ -21,16 +21,23 @@ <h1 class="page-header">Dashboard</h1>
             </div>
             <div class="col-sm-5">
               <h3> Events </h3>
-              <div class = "event" ng-repeat="event in events" ng-click='goToEvent(event)'>
+              <div class = "event" ng-repeat="event in events">
                 <div class="event-card">
                   <div class="event-title">
-                    {{event.info.title}}
+                    {{event.title}}
                   </div>
                   <div class="event-time">
-                    Start: {{event.info.times[0].start}}
+                    Start: {{event.times[0].start}}
```

Comment:
good points. Since there must be an end time anyways, I think including it would be a good idea.